docs/proposals-accepted/202304-query-path-tenancy.md

```diff
@@ -0,0 +1,139 @@
+---
+type: proposal
+title: Tenancy awareness in query path
+status: accepted
+owner: douglascamata
+menu: proposals-accepted
+---
+
+## Tenancy awareness in query path
+
+* **Owners:**
+    * [@douglascamata](https://github.com/douglascamata)
+
+* **Related Tickets:**
+    * [Querier: Added native multi-tenancy support](https://github.com/thanos-io/thanos/pull/4141).
+    * [Proposal: Native Multi-tenancy Support](https://github.com/thanos-io/thanos/pull/4055)
+
+* **Other docs:**
+    * [Multi-tenancy in Thanos](https://thanos.io/tip/operating/multi-tenancy.md/)
+
+<!--
```

Comment:
Can we remove these comments? Causes no diff in rendering, but makes it difficult to review. 🙂